offline.js

```diff
@@ -22,6 +22,30 @@ export const loadTextOffline = async function(ref, context, versions, fallbackOn
     return {result: textFromRefData(processed)};
 };
 
+export const getAllTranslations = async function (ref, versions) {
```

Comment:
Also, `versions` doesn't seem to make sense as a parameter here. If I understand correctly, it's used to avoid a call to `getOfflineVersionObjectsAvailable` which doesn't seem to be a very expensive function. By passing in `versions` it makes the control flow confusing. Is versions an internal variable or is it an input?

Reply:
i'm adding offline for we have it in all functions, but in principle i think we should remove all the offline from functions in module that named offline, if there is no a good reason.